compiler/qsc_linter/src/linter/ast.rs

```diff
@@ -62,7 +62,6 @@ pub(crate) trait AstLintPass {
     fn check_stmt(&self, _stmt: &Stmt, _buffer: &mut Vec<Lint>) {}
     fn check_ty(&self, _ty: &Ty, _buffer: &mut Vec<Lint>) {}
     fn check_ty_def(&self, _ty_def: &TyDef, _buffer: &mut Vec<Lint>) {}
```

Comment:
It occurs to me that one side effect of removing visibility from the AST is that we won't be able to add a lint warning in the future to tell people it's unnecessary and should be removed. Is the possibility of a future lint enough of a reason to keep it in?